portality/templates/admin/edit_article_metadata.html

```diff
@@ -0,0 +1,161 @@
+{% extends "admin/admin_base.html" %}
+
+{% block content %}
+{% set object_type = 'article' %}
+{% set heading_object_type = 'Article' %}
+{% set form_context = form_context %}
+
+<div class="row" style="margin-bottom: 20px">
+    <div class="col-md-9">
+        <h1>{{ heading_object_type }}: {{form_context.source.bibjson().title}}</h1>
+        {% if form_context.source.last_updated %}
+          <h4>Last updated: {{form_context.source.last_updated}}</h4>
+        {% endif %}
+        {% if form_context.source.current_journal %}
+            <h4 style="margin-top: 1em;"><a href="{{ url_for('doaj.journal_readonly', journal_id=form_context.source.current_journal) }}" target="_blank">View associated journal</a></h4>
+        {% endif %}
```

Comment:
I think it is useful. It gives the admin clear information. Easy to remove if you think it would be better to do so.

![image](https://user-images.githubusercontent.com/8298769/74835567-8c362a00-5315-11ea-8b95-9564de27044d.png)